### ISSUE TITLE
can't reindex

### ISSUE THREAD

**Issue by jcoyne** [curationexperts/goldenseal#106]
``` ruby
gf = Text.find('1831cj937')
gf.update_index
TypeError: no implicit conversion from nil to integer
    from /opt/goldenseal/shared/bundle/ruby/2.2.0/gems/activefedora-aggregation-0.4.0/lib/active_fedora/aggregation/association.rb:55:in `delete_at'
    from /opt/goldenseal/shared/bundle/ruby/2.2.0/gems/activefedora-aggregation-0.4.0/lib/active_fedora/aggregation/association.rb:55:in `create_linked_list'
    from /opt/goldenseal/shared/bundle/ruby/2.2.0/gems/activefedora-aggregation-0.4.0/lib/active_fedora/aggregation/association.rb:57:in `create_linked_list'
    from /opt/goldenseal/shared/bundle/ruby/2.2.0/gems/activefedora-aggregation-0.4.0/lib/active_fedora/aggregation/association.rb:57:in `create_linked_list'
    from /opt/goldenseal/shared/bundle/ruby/2.2.0/gems/activefedora-aggregation-0.4.0/lib/active_fedora/aggregation/association.rb:57:in `create_linked_list'
    from /opt/goldenseal/shared/bundle/ruby/2.2.0/gems/activefedora-aggregation-0.4.0/lib/active_fedora/aggregation/association.rb:57:in `create_linked_list'
    from /opt/goldenseal/shared/bundle/ruby/2.2.0/gems/activefedora-aggregation-0.4.0/lib/active_fedora/aggregation/association.rb:57:in `create_linked_list'
    from /opt/goldenseal/shared/bundle/ruby/2.2.0/gems/activefedora-aggregation-0.4.0/lib/active_fedora/aggregation/association.rb:57:in `create_linked_list'
    from /opt/goldenseal/shared/bundle/ruby/2.2.0/gems/activefedora-aggregation-0.4.0/lib/active_fedora/aggregation/association.rb:25:in `ids_reader'
    from /opt/goldenseal/shared/bundle/ruby/2.2.0/gems/activefedora-aggregation-0.4.0/lib/active_fedora/aggregation/builder.rb:18:in `block in define_readers'
    from /opt/goldenseal/shared/bundle/ruby/2.2.0/bundler/gems/curation_concerns-b79bd9ee71f2/curation_concerns-models/app/services/curation_concerns/generic_work_indexing_service.rb:10:in `block in generate_solr_document'
    from /opt/goldenseal/shared/bundle/ruby/2.2.0/bundler/gems/curation_concerns-b79bd9ee71f2/curation_concerns-models/app/services/curation_concerns/generic_work_indexing_service.rb:5:in `tap'
    from /opt/goldenseal/shared/bundle/ruby/2.2.0/bundler/gems/curation_concerns-b79bd9ee71f2/curation_concerns-models/app/services/curation_concerns/generic_work_indexing_service.rb:5:in `generate_solr_document'
    from /opt/goldenseal/releases/20150904225000/app/indexers/text_indexer.rb:5:in `generate_solr_document'
    from /opt/goldenseal/shared/bundle/ruby/2.2.0/gems/active-fedora-9.3.0/lib/active_fedora/indexing.rb:15:in `to_solr'
    from /opt/goldenseal/shared/bundle/ruby/2.2.0/bundler/gems/curation_concerns-b79bd9ee71f2/curation_concerns-models/app/models/concerns/curation_concerns/human_readable_type.rb:15:in `to_solr'
    from /opt/goldenseal/shared/bundle/ruby/2.2.0/gems/hydra-access-controls-9.2.2/app/models/concerns/hydra/access_controls/permissions.rb:15:in `to_solr'
    from /opt/goldenseal/shared/bundle/ruby/2.2.0/bundler/gems/curation_concerns-b79bd9ee71f2/curation_concerns-models/app/models/concerns/curation_concerns/has_representative.rb:9:in `to_solr'
    from /opt/goldenseal/shared/bundle/ruby/2.2.0/gems/hydra-access-controls-9.2.2/app/models/concerns/hydra/access_controls/embargoable.rb:37:in `to_solr'
    from /opt/goldenseal/shared/bundle/ruby/2.2.0/gems/active-fedora-9.3.0/lib/active_fedora/indexing.rb:24:in `update_index'
```

**Comment by jcoyne** [comment]
Also when viewing http://wustl.curationexperts.com/concern/images/j9602060k/edit

```
ActionView::Template::Error (no implicit conversion from nil to integer):
    1: <% if curation_concern.persisted? && curation_concern.member_ids.present? %>
    2:   <div class="row">
    3:     <div class="col-md-12">
    4:       <fieldset id="representative-image">
  activefedora-aggregation (0.4.0) lib/active_fedora/aggregation/association.rb:55:in `delete_at'
  activefedora-aggregation (0.4.0) lib/active_fedora/aggregation/association.rb:55:in `create_linked_list'
  activefedora-aggregation (0.4.0) lib/active_fedora/aggregation/association.rb:57:in `create_linked_list'
  activefedora-aggregation (0.4.0) lib/active_fedora/aggregation/association.rb:57:in `create_linked_list'
  activefedora-aggregation (0.4.0) lib/active_fedora/aggregation/association.rb:25:in `ids_reader'
  activefedora-aggregation (0.4.0) lib/active_fedora/aggregation/builder.rb:18:in `block in define_readers'
  /opt/goldenseal/shared/bundle/ruby/2.2.0/bundler/gems/curation_concerns-4d873a859105/app/views/curation_concerns/base/_form_representative_image.html.erb:1:in `__opt_goldenseal_shared_bundle_ruby_______bundler_gems_curation_concerns__d___a_______app_views_curation_concerns_base__form_representative_image_html_erb___3859964538396410999_70250837766900'
```

**Comment by jcoyne** [comment]
Fixed in ActiveFedora-Aggregations 0.4.1
